tests/course/test_for_review_course.py

```diff
@@ -23,3 +23,18 @@ def test_student_reviewer(self, persist_review):
     def test_get_number_of_review_of_course(self, persist_review, persist_review_number_two):
         result = Review.objects.get_number_of_review_of_course(persist_review.course)
         assert result == 2
+
+    def test_check_if_review_exist_by_student(self, persist_review, persist_second_course):
```

Comment:
This should be a parameterized test instead of duplicating the code block with a different parameter.

Reply:
Can you explain how to do it with fixture and parametrize? I don't see anyone using it in our team.

Reply:
You can use a `fixture factory`, where you create a fixture that get inputs, then you parametrize just the values that change in the object from test to test, and create the fixture on-the-go. Its been done in this repo in some sections, I can't remember exactly where...

Reply:
https://github.com/redhat-beyond/HighTeach/pull/75

Reply:
I reworte the test so it will use the fixture factory, how can I utlize parametrize in this test?
The flow of the new test is this :
It is a asserted that the user does not have a review for course (What return from the function is none)
Create review for user for a said course 
Asserted that the user does have a review for course (What return from the function is not none)